Mathlib/Init/Data/Nat/Basic.lean

```diff
@@ -13,6 +13,8 @@ set_option linter.deprecated false
 
 theorem add_one_pos (n : ℕ) : 0 < n + 1 := succ_pos n
 
+@[simp] theorem default_eq : default = 0 := rfl
```

Comment:
Sorry, I was suggesting the name of an existing lemma but apparently said lemma doesn't exist!
```suggestion
@[simp] theorem default_eq_zero : default = 0 := rfl
```
Can you also move this to `Data.Nat.Defs`? Nothing more should be going under `Init`

Reply:
Done. In `Data.Nat.Defs` I wasn't quite sure where to put it, because it didn't quite fit under any of the sections, so I put it near the top. Happy to move it around if we find a better place for it.